analysis_options.yaml

```diff
@@ -0,0 +1,34 @@
+analyzer:
+  strong-mode: true
+  errors:
+    unused_import: error
+    unused_local_variable: error
+    dead_code: error
```

Comment:
Why does Google3 config control which hints are upgraded in third-party packages?